packages/vite/src/client/overlay.ts

```diff
@@ -1,8 +1,17 @@
 import type { ErrorPayload } from 'types/hmrPayload'
 
+// set :host styles to make playwright detect the element as visible
```

Comment:
Another way is to set `{ state: 'attached' }` but that will still might break ecosystem tests that relies on this.